app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -25,28 +25,30 @@ class MapViewCustomizationActivity : AppCompatActivity() {
   }
 
   private fun configureMapViewFromCode() {
-    val mapboxMapOptions = MapboxMapOptions(this).apply {
+    // set map options
+    val mapOptions = MapOptions.Builder()
+      .constrainMode(ConstrainMode.HEIGHT_ONLY)
+      .glyphsRasterizationOptions(
+        GlyphsRasterizationOptions.Builder()
+          .rasterizationMode(GlyphsRasterizationMode.IDEOGRAPHS_RASTERIZED_LOCALLY)
+          .build()
+      )
+      .build()
+
+    // set token and cache size for this particular map view
+    val resourceOptions = ResourceOptions.Builder()
+      .accessToken(getString(R.string.mapbox_access_token))
+      .cacheSize(75_000L)
+      .build()
+
+    val mapboxMapOptions = MapInitOptions(this, resourceOptions, mapOptions).apply {
```

Comment:
we can make cameraOptions and textureView as part of `MapInitOptions`'s constructor.

Reply:
Right